packages/api/models/schemas.js

```diff
@@ -36,11 +36,13 @@ module.exports.asyncOperation = {
   title: 'AsyncOperation Object',
   description: 'Cumulus API AsyncOperation Table schema',
   type: 'object',
-  required: ['createdAt', 'id', 'status', 'updatedAt'],
+  required: ['createdAt', 'id', 'status', 'updatedAt', 'description', 'operationType'],
   additionalProperties: false,
   properties: {
     createdAt: { type: 'integer' },
     id: { type: 'string' },
+    description: { type: 'string' },
+    operationType: { type: 'string' },
```

Comment:
I'm wondering if this should be an enumeration so things don't deviate and then we could keep it to:
`Bulk granule`,
`ES Index`,
`Kinesis Replay`

Reply:
changed